src/db_impl.cc

```diff
@@ -378,10 +388,6 @@ Status TitanDBImpl::DropColumnFamilies(
   Status s = db_impl_->DropColumnFamilies(handles);
   if (s.ok()) {
     MutexLock l(&mutex_);
-    for (auto cf_id : column_families) {
```

Comment:
why remove this

Reply:
`base_table_factory_` and `titan_table_factory_` is merged into `cf_info_` and they get erased together lines below.